Wrapping/Python/CMakeLists.txt

```diff
@@ -19,6 +35,24 @@ mark_as_advanced( SimpleITK_PYTHON_EGG )
 option ( SimpleITK_PYTHON_WHEEL "Add building of python wheels to the dist target." ON )
 mark_as_advanced( SimpleITK_PYTHON_WHEEL )
 
+if(SWIG_VERSION VERSION_GREATER_EQUAL "4.2.0" AND
+        Python_VERSION VERSION_GREATER_EQUAL "3.11.0") # CMake atleast 3.26
+  set(_SimpleITK_PYTHON_USE_LIMITED_API_DEFAULT ON)
+else ()
+  set(_SimpleITK_PYTHON_USE_LIMITED_API_DEFAULT OFF)
+endif()
+
+option( SimpleITK_PYTHON_USE_LIMITED_API "Use Python limited API, for minor version compatibility." ${_SimpleITK_PYTHON_USE_LIMITED_API_DEFAULT} )
+mark_as_advanced( SimpleITK_PYTHON_USE_LIMITED_API )
+
+if (SimpleITK_PYTHON_USE_LIMITED_API AND NOT _SimpleITK_PYTHON_USE_LIMITED_API_DEFAULT)
+  message(WARNING "Python limited API is not supported for this Python version and SWIG version.")
+  message(WARNING "SWIG_VERSION: ${SWIG_VERSION}")
+  message(WARNING "Python_VERSION: ${Python_VERSION}")
+  message(WARNING "Development.SABIModule_FOUND: ${Development.SABIModule_FOUND}")
+  set(SimpleITK_PYTHON_USE_LIMITED_API OFF)
```

Comment:
Please add a message specifying the minimal version requirements for SWIG and Python. 

Not sure about this, should this be an error? User turned SimpleITK_PYTHON_USE_LIMITED_API on but package versions aren't valid.

Reply:
done